crates/cli/tests/install.rs

```diff
@@ -131,3 +132,32 @@ fn should_install_index_files() {
 
     drop((root, mock_instance)); // cleanup
 }
+
+#[test]
+fn should_install_duplicated_dependencies() {
+    let CommandTempCwd { pacquet, root, workspace, .. } =
+        CommandTempCwd::init().add_mocked_registry();
+
+    eprintln!("Creating package.json...");
+    let manifest_path = workspace.join("package.json");
+    let package_json_content = serde_json::json!({
+        "dependencies": {
```

Comment:
Before rebase the branch main, i.e. without mock-registry, the test passed, but now it doesn't seem to be able to install `express`, so it will take a bit more time to see what's going on.